### DISCUSSION TITLE
by using curl to get the search result, the attribution of 'raduis' did not work

### DISCUSSION THREAD

**Discussion by syj121** [milvus-io/milvus#27843] [Q&A and General discussion] [answered]
curl --location --request POST 'localhost:19530/v1/vector/search' \
--header 'Content-Type: application/json' \
--header 'Cookie: BetterErrors-2.9.1-CSRF-Token=f0b8eacb-04ff-4ae6-94fe-ea7b9f7bbdd1' \
--data-raw '{
    "collectionName": "xxx",
    "outputFields": [
        "id",
        "a",
        "b"
    ],
    "params": {
        "raduis": 0.1
    },
    "vector": [1,2,3]
}'

the result with distance ganter 0.1  returned. following is the result

<img width="774" alt="image" src="https://github.com/milvus-io/milvus/assets/16488342/3f9eeea3-e2dc-44ac-8e3b-bcf48ed4f3a5">

**Comment by xiaofan-luan** [comment, marked as answer]
this might because of restful API don't support range search.
intersted in contributing on this?
